pkg/machine/default.go

```diff
@@ -0,0 +1,154 @@
+package machine
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/image/v5/types"
+	"github.com/containers/podman/v4/pkg/machine/ocipull"
+	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/sirupsen/logrus"
+)
+
+type Default struct {
```

Comment:
Default is a really strange name, it really make no sense as name here in the machine package.
Maybe name it `OCIPull`?

Reply:
Also why does this live inside the machine package at all? Shouldn't this stuff also be part of ocipull.

Reply:
circular deps ... ;(